lib/reply.js

```diff
@@ -195,6 +197,12 @@ Reply.prototype.headers = function (headers) {
   return this
 }
 
+Reply.prototype.skipSerialization = function () {
+  this[kReplyskipSerialization] = true
+  this.context[kReplyskipSerialization] = true
```

Comment:
Unfortunately this won't work. The context is shared between multiple execution of the handler and multiple reply objects. Can you remove it? The alternative is to move this to the Context itself, and set this option during the route creation.

Reply:
You told about this.context? What about just this? Will it work If i pass from this[kReplyskipSerialization] to serialize() method as optional params?

Reply:
I would actually change the serialize function to receive the Reply object instead.

Reply:
Changed it to pass this param via `this` of reply object. Added test to check that `skipSerialization` call doesn't affect next responses.